internal/sync/configs/olm-config.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- name: olm
```

Comment:
Please use a name that better reflects the api being cloned

Reply:
it's olm - operator-lifecycle-manager. OLM is common name for this module. More folks now it as olm and not as olm - operator-lifecycle-manager.

Reply:
I mean it doesn't give any info regarding what you're actually cloning. I think it should be something like olm-v1alpha1